### DISCUSSION TITLE
Full STACK web sites or web plateforms

### DISCUSSION THREAD

**Discussion by network1618033** [AntonOsika/gpt-engineer#450] [Q&A]
Can we build full STACK websites with this. Frontend design with backend ?

**Comment by LouisWhit** [comment]
No. At best, I can only get it to roughly layout project files.

**Reply by vVv-Keys** [reply]
I been working on real time interactions and real-time context awareness with some tailored guidance for a while now to advance a few different things in the field. But @network1618033 to make full STACK websites takes a lot of HANDS ON DEVELOPMENT. GPT engineer is indeed a nifty scripted encyclopedia especially when you give it the right philosphy and test etc.. but that's all it is until people go about making it have a shell of its own to execute different langs in a sandbox or enclosed testing environment in REAL-time. ACESS to the external resources like a resciever/transmittor to keep information and data stored properly with all external services. A complete UI of some sort like a future ROBOT at that point so imagine the JETSONS and ROSIE (if you know what this is I might be old)..

Then ERROR HANDLING/DEBUGGING is a very HANDS ON THING FOR STACK DEVELOPERS currently there is no way to give the typescript arch "HANDS-ON" type feature persay to be able to acesss the proper tools it would need to handle the errors and debug properly...

I have been working on a few of these things myself to hopefully further advance what we know as GPT to more of a sentient type of AI model than just a simple TS... but these are tasks what most would call crazy or "BEYOND THE SCOPE" currently for AI lang models.. including the techincal challenges to implement the capabilities properly without exploits or CRUCIAL robust errors etc...

hope this helps ya'll though - have a blessed week!
